Outdated mention of install_crds makefile rule in README

### Description
A mention of the `install_crds` makefile rule exists in the README, however it has since been removed.

### How To Reproduce
Steps to reproduce the behavior:
1. Clone devworkspace-operator
2. Go to the root of the git repository
3. Do a `make install_crds`
4. See makefile error: `make: *** No rule to make target 'install_crds'.  Stop.`

### Expected behavior
The make rule runs and installs the CRDs on cluster.

### Additional context
I assume mention of the `install_crds` makefile rule should be removed from the README. I can submit a quick PR to do this.

--- a/README.md
+++ b/README.md
@@ -104,7 +104,6 @@ Some of the rules supported by the makefile:
 | docker | build and push docker image |
 | install | install controller to cluster |
 | restart | restart cluster controller deployment |
-| install_crds | update CRDs on cluster |
 | install_cert_manager | installs the cert-manager to the cluster (only required for Kubernetes) |
 | uninstall | delete controller namespace `devworkspace-controller` and remove CRDs from cluster |
 | help | print all rules and variables |

